### ISSUE TITLE
Team 9 - P5 - set up round - modal does not close for waiting

### ISSUE THREAD

**Issue by tsmith1644** [cs340tabyu/cs340Winter2017#117]
Modal for building settlements and roads does not close while the player should be waiting on other players. It just doesn't allow building anywhere.

**Comment by cojoco** [comment]
Hey, is this issue for team 7 or team 9? If it's team 9, can you please change the label to say so (on this and any other issues you logged for team 9)?

Done, the other two I did were on your code, it just wasn't clear what team we were supposed to be testing

**Comment by UnoriginalElephant** [comment]
duplicate of #245
